File: Homeworks/hw1_java/src/main/java/com/pascal/homeworks/MainActivity.java
package com.pascal.homeworks;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity implements View.OnClickListener{
    private Button btnToSecAct;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnToSecAct = findViewById(R.id.fvd_btn);
        btnToSecAct.setOnClickListener(this);
    }

    @Override
    public void onClick(View v) {
        switch (v.getId()) {
            case R.id.fvd_btn:
                Intent intent = new Intent(this, SecondActivity.class);
                startActivity(intent);
                finish();
                break;
        }
    }
}

File: Homeworks/hw1_kotlin/src/main/java/com/pascal/hw1_kotlin/MainActivity.kt
package com.pascal.hw1_kotlin

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    private lateinit var btnToSecAct: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnToSecAct = findViewById(R.id.fvd_btn)
        btnToSecAct.setOnClickListener { view: View ->
            val intent = Intent(this, SecondActivity::class.java)
            startActivity(intent)
            finish()
        }
    }
}